### PR TITLE
Edits for release

### PR DESCRIPTION
**Pull request by nebelsprachen**
_Labels:_
* enterprise-4.1
* enterprise-4.2
--------------------------------------------------------------------------
Minor edits for Serverless release

### CODE REVIEW

File: modules/serverless-document-attributes.adoc [hunk 1]
```diff
@@ -10,6 +10,11 @@
 :ServerlessProductName: OpenShift Serverless
 :ServerlessProductShortName: Serverless
 :ServerlessOperatorName: OpenShift Serverless Operator
+:product-title: OpenShift Container Platform
+:ProductName: Red Hat OpenShift Service Mesh
```
Comment (vikram-redhat) [inline modules/serverless-document-attributes.adoc]: @joaedwar a few typos here.

File: serverless/installing-openshift-serverless.adoc [hunk 2]
```diff
@@ -25,13 +25,18 @@ include::modules/serverless-installing-service-mesh-member-roll.adoc[leveloffset
 [id="installing-serverless-operator_{context}"]
 == Installing the OpenShift Serverless Operator
 
-The OpenShift Serverless Operator can be installed using the {product-title} instructions for installing Operators.
+The OpenShift Serverless Operator can be installed using the {product-title} instructions for installing Operators.
```
Comment (vikram-redhat) [inline serverless/installing-openshift-serverless.adoc]: This should be replaced with {ServerlessProductName}?

File: modules/serverless-rn-new-features.adoc [hunk 3]
```diff
@@ -18,5 +18,5 @@ may have changed.
 
 == New features in Technology Preview 1
 
-This release of {ProductName} introduces the OpenShift Serverless Operator,
+This release introduces the OpenShift Serverless Operator,
```
Comment (vikram-redhat) [inline modules/serverless-rn-new-features.adoc]: Should be replaced with the attribute?

File: serverless/installing-openshift-serverless.adoc [hunk 4]
```diff
@@ -25,13 +25,18 @@ include::modules/serverless-installing-service-mesh-member-roll.adoc[leveloffset
...
 
-The OpenShift Serverless Operator can be installed using the {product-title} instructions for installing Operators.
+The OpenShift Serverless Operator can be installed using the {product-title} instructions for installing Operators. 
 
 You can install the OpenShift Serverless Operator in the host cluster by following the {product-title} instructions on installing an Operator.
 
 [NOTE]
 ====
-The Serverless Operator only works for OpenShift versions 4.1.13 and later.
+The Serverless Operator only works for OpenShift versions 4.1.13 and later.
```
Comment (vikram-redhat) [inline serverless/installing-openshift-serverless.adoc]: Same - should be replaced with the attribute?

**Comment by vikram-redhat** [comment]
@openshift/team-documentation PTAL.

**Review by abrennan89 — CHANGES_REQUESTED** [review]

**Pull request merged**